includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -94,8 +94,8 @@ public function read( &$coupon ) {
 		$coupon->set_props( array(
 			'code'                        => $post_object->post_title,
 			'description'                 => $post_object->post_excerpt,
-			'date_created'                => strtotime( $post_object->post_date_gmt ),
-			'date_modified'               => strtotime( $post_object->post_modified_gmt ),
+			'date_created'                => 0 < $post_object->post_date_gmt ? wc_string_to_timestamp( $post_object->post_date_gmt ) : null,
```

Comment:
Well I'll be darned. :) 

After more testing, it looks like `0 < '0000-00-00 00:00:00'` is false in PHP and `0 < '2017-03-23 18:48:11'` is true. You're absolutely correct; this works properly. I misunderstood what happens when those time strings are cast to an integer.

Everything looks good to me, then. 👍